solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/RestaurantCellView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  SampleComponentView.swift
+//  DeliveryAppChallenge
+//
+//  Created by Rodrigo Borges on 04/02/22.
+//
+
+import UIKit
+
+struct RestaurantCellViewConfiguration {
+    let restaurantName: String
+    let restaurantType: String
+    let restaurantImage: String
+}
+
+class RestaurantCellView: UIView {
+    
+    private lazy var restaurantName: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 15)
```

Comment:
Uma dica de syntax sugar, é que você não precisa de fato especificar o `UIFont` para acessar os métodos e propriedades dele, como o `label.font` já é do tipo `UIFont`, o compilador consegue já inferir as propriedades.
```suggestion
        label.font = .boldSystemFont(ofSize: 15)
```